search/templates/search/search.html

```diff
@@ -6,19 +6,98 @@
 {% endblock %}
 
 {% block content %}
-<div>
-    <h1>Hey {{ loggedUser.first_name }}!</h1>
```

Comment:
You dont have to send 'loggedUser' in context,
django will automatically send the request user under the name 'user'.
for example this line can be switched to `<h1>Hey {{ user.first_name }}!</h1>` without sending special context.